ops/ansible/roles/bfd-pipeline/tasks/main.yml

```diff
@@ -45,62 +45,17 @@
   become: true
   when: data_pipeline_tmp_dir is defined and stat_tmp.stat.exists == false
 
-# If the app's JAR is in S3, grab it from there.
```

Comment:
Removing this is great, as in CCS it's taken care of in platinum build.

Reply:
Yeah. Only the old ETL benchmarks were using it, and I'd rather rework them to use AMIs (if I ever need them again).